webapp/data_structures/MAXMIN.js

```diff
@@ -39,14 +37,13 @@ function MAXMIN()
     MAXMIN.prototype.set = function MAXMIN() {
     this.mm = parseFloat((arguments[0]["mm"]))
     this.nin = parseFloat((arguments[0]["nin"]))
-    this.tag = arguments[0]["tag"]
     if((this.nin!=1)&&(this.nin!=2)){
                 alert("Wrong number of inputs, only 1 and 2 allowed");
                 throw "incorrect";
     }
     this.zcr = parseFloat((arguments[0]["zcr"]))
     if(this.zcr != 0){
-        this.zcr = 1
+        this.zcr = -1
```

Comment:
```suggestion
        this.zcr = 1;
```
The earlier change was correct.